src/get-partition-key.ts

```diff
@@ -0,0 +1,19 @@
+import Collection from "./account/collection";
+
+/**
+ * Assumes only 1 partition key path in the following format:
+ *  - paths: ["/<partitionKey>"]
+ * @returns "<partitionKey>" string or null
+ */
+export default function getPartitionKey(collection: Collection) {
+  // eslint-disable-next-line
+  const partitionKey = collection.partitionKeyRanges._parent._data.partitionKey;
```

Comment:
```suggestion
  const { partitionKey } = collection.read() || {};
```
It should be accessible like this.